src/Management/test/Endpoint.Test/Actuators/Health/HealthEndpointOptionsTest.cs

```diff
@@ -74,17 +81,4 @@ public void Constructor_BindsRoleCorrectly()
         Assert.Equal(ClaimTypes.Role, options.Claim.Type);
         Assert.Equal("role-claim-value", options.Claim.Value);
     }
-
-    [Fact]
-    public void CanClearDefaultGroups()
```

Comment:
From what I understand, readiness/liveness contributors are not added automatically anymore. However, when they _are_ added, their groups are automatically created. We still need a way to configure not creating any groups.